index.html

```diff
@@ -140,7 +140,35 @@ <h1 class="title_post">Multi-Post Stories</h1>
   
       <section class="button">See Project</section>
     </section>
-
+    <section id="contact-me">
+      <header>
+        <h2>CONTACT US</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+        </p>
+      </header>
+        <form action="https://formspree.io/f/xrgdbeyr" method="post">
+          <fieldset>
+          <input type="text" id="name" name="Name" placeholder="Text Input" required
+          maxlength="30"
+          />
+          <input type="email" id="email" name="Email" placeholder="jeremy@gmail.com" required
+          />
+          <textarea name="Text Area" id="textarea" placeholder="Enter text here" required></textarea>
```

Comment:
- [ ] Kindly add validations to your textarea to make it more professional. this textarea should have` maxlength="500" `attribute. you got this 👍